DEVNOTES.txt

```diff
@@ -12,6 +12,9 @@ With LGPL dependencies:
 With LGPL dependencies and Scala 2.10:
   mvn clean package -DskipTests -Prelease,lgpl -Dignite.edition=fabric-lgpl -Dscala-2.10
 
+With LGPL dependencies and Scala 2.10 and Ignite-Yardstick:
```

Comment:
Thank you, I think you are right. 
Anton had suggested this command:
mvn clean package -DskipTests -Prelease,yardstick
so I've changed the lines in such way.